src/lib/handlePageRequest.ts

```diff
@@ -38,29 +37,13 @@ export const handlePageRequest = (
 ): GetServerSideProps =>
   withCookieRedirect(
     withContainer(async (context: BeaconsGetServerSidePropsContext) => {
-      const {
-        getCachedRegistration,
-        saveCachedRegistration,
-        authenticateUser,
-      } = context.container;
+      const { authenticateUser } = context.container;
 
       await authenticateUser(context);
 
       const beaconsContext: BeaconsContext =
         await decorateGetServerSidePropsContext(context);
 
-      const registration: Registration = await getCachedRegistration(
-        retrieveUserFormSubmissionId(context)
-      );
-
-      const useIndexDoesNotExist =
-        beaconsContext.useIndex >
-        registration.getRegistration().uses.length - 1;
-      if (useIndexDoesNotExist) {
-        registration.createUse();
-        await saveCachedRegistration(beaconsContext.submissionId, registration);
-      }
```

Comment:
Here's the culprit.

![image](https://user-images.githubusercontent.com/54271433/126365415-78f8dd01-2df3-48a7-a7e4-55735ff9ecdd.png)